sdk/servicefabricmanagedclusters/Microsoft.Azure.Management.ServiceFabricManagedClusters/src/Generated/Models/ErrorModelException.cs

```diff
@@ -0,0 +1,61 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.ServiceFabricManagedClusters.Models
```

Comment:
So, this is occurring because your spec is not referencing the common ARM error model (https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json#L277).  Using a custom error like this, rather than the exception form the management library will make it more difficult to handle errors uniformly across RP sdks for the customer

Reply:
Ok. I understand thanks! we will fix this on the next swagger update